coala_quickstart/generation/Project.py

```diff
@@ -50,6 +52,22 @@ def language_percentage(file_paths):
         if ext in exts:
             for lang in exts[ext]:
                 results[lang] += delta
+        elif not ext:
+            with open(file_path, 'r') as data:
+                hashbang = data.readline()
+                if(re.match(
+                        '/(^#![(a-z)|\/]*[ ][a-z]*)|(^#![(a-z)|\/]*)/',
```

Comment:
Maybe have this regex somewhere in the Constants file since it's reused in Utilities?